kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -155,6 +155,15 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
     SessionHandle(HandleIdentifier(UUID.fromString(batchId), STATIC_BATCH_SECRET_UUID), protocol)
   }
 
+  def getBatchSessionList(batchType: String, from: Int, size: Int): Seq[Session] = {
+    allSessions().filter {
+      case batchSession: KyuubiBatchSessionImpl =>
+        batchSession.batchJobSubmissionOp.batchType.equals(batchType)
```

Comment:
It seems that now the batchType is case-insensitive and not be normalized.

```
batchType.equalsIgnoreCase(batchType)
```

Reply:
done